orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/DelegatingOortService.java

```diff
@@ -36,8 +36,13 @@ public Response getCluster(String app, String account, String cluster, String cl
   }
 
   @Override
-  public Response getServerGroup(String app, String account, String cluster, String serverGroup, String region, String cloudProvider) {
-    return getService().getServerGroup(app, account, cluster, serverGroup, region, cloudProvider);
+  public Response getServerGroupFromCluster(String app, String account, String cluster, String serverGroup, String region, String cloudProvider) {
```

Comment:
`getServerGroupFromCluster()` was previously known as `getServerGroup()` and is rather inefficient (loads the cluster and then filters).

I opt'd to not chase down callers and see which could be switched to the more optimal `getServerGroup()`, this can be done later. Most should be easily converted but there are a couple situations where `region` is not available.